.github/workflows/cli-coexistence_tests.yaml

```diff
@@ -67,5 +57,11 @@ jobs:
       - name: Tests Results Summary
         if: always()
         run: |
-          TEST_RESULTS_MD=$(./hack/scripts/process-ginkgo-test-results.sh test/e2e/coexistence)
+          if [ -d "test/e2e/coexistence/testresults" ]; then
+          echo "Coexistence Test Results exists"
+          else
+            echo "Coexistence Test Results does not exist"
+          fi
```

Comment:
Is the above for debugging or is it still valid?
If valid, the indentation is wrong.

Reply:
yes This is for testing purpose. removed the logs